app/models/katello/glue/candlepin/pool.rb

```diff
@@ -135,7 +135,7 @@ def import_data
       end
 
       def hosts
-        entitlements = Resources::Candlepin::Pool.entitlements(self.cp_id)
+        entitlements = Resources::Candlepin::Pool.entitlements(self.cp_id, ["consumer"])
```

Comment:
if you change this to 'consumer.uuid' you get even less information:

[{"consumer"=>{"uuid"=>"9680131b-b631-4108-953b-a53d0a21cac1"}}]
vs
[{"consumer"=>{"id"=>"4028f9515c87c485015c8a592eee000a", "uuid"=>"9680131b-b631-4108-953b-a53d0a21cac1", "name"=>"balmora.redhat.com", "href"=>"/consumers/9680131b-b631-4108-953b-a53d0a21cac1"}}]